assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js

```diff
@@ -58,14 +55,20 @@ const CategoryPillInnerContainer = styled.div`
 // Flips the button upside down when expanded;
 // Important: the visibily is 'inherit' when props.visible because otherwise
 // it gets shown even when the provider is not the selectedProvider!
-const ExpandButton = styled(ArrowDown).attrs(({ isExpanded }) => ({
-  'aria-label': isExpanded
-    ? __('Collapse Categories', 'web-stories')
-    : __('Expand Categories', 'web-stories'),
-}))`
-  ${(props) => props.isExpanded && 'transform: matrix(1, 0, 0, -1, 0, 0);'};
-  visibility: ${(props) => (props.visible ? 'inherit' : 'hidden')};
+const ExpandButton = styled(ArrowDown)`
+  display: flex;
+  position: absolute;
+  bottom: -12px;
+  background: ${({ theme }) => theme.colors.fg.gray16};
+  max-height: none;
+  width: 32px;
```

Comment:
Width and height are also params for `ArrowDown` directly.

Same goes for `visibility` which could be used via `isHidden`.

Reply:
Actually this couldn't really be done much better - both because `visibility` has to be `inherit` and not `visible` and as it makes sense to have width and height defined here rather than elsewhere.